_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -47816,6 +47816,7 @@
 /obj/effect/turf_decal/stripes/line{
 	dir = 9
 	},
+/obj/structure/closet/secure_closet/miner,
```

Comment:
That is correct. The mining locker in ALL Aux Bases should be unlocked. It is intended to let other crew experience Lavaland, OR be a second base the miners can use if a Drake decided to eat the primary base.